index.html

```diff
@@ -438,6 +438,41 @@ <h2>Data Model</h2>
 <a>verification methods</a>.
       </p>
 
+      <p>
+All of the data model properties and types in this specification map to URLs.
+The vocabulary where these URLs are defined is the [[?SECURITY-VOCABULARY]]. The
+explicit mechanism that is used to perform this mapping in a secured document is
+the `@context` property.
+      </p>
+      <p>
+When processing a document in a JSON-LD environment, the mapping mechanism is
+defined by JSON-LD [[JSON-LD11]]. When processing a document in a non-JSON-LD
+environment, the same mechanism is used to ensure that the terms have the same
+interpretation regardless of format, but with a simpler processing rule:
+      </p>
+
+      <p>
+Ensure that all values associated with a `@context` property are in the expected
+order, the context files match documented cryptographic hashes for each file,
```

Comment:
"Expected order": at this point it is not clear what you mean. I _presume_ you want to say something liked "the expected order as defined by the domain expert" or something like that.

Reply:
Hrm, no, it's only meant to be the following paragraph. The part about `@context` injection is a separate rule on what to do if the `@context` value is missing by accident. 

I can see how the two normative statements, one right after the other, combined with the colon above would cause confusion to a reader. I'll try to fix up the grammar/flow in an update to this PR.

Reply:
Fixed in 02deef295eb69fcca67c38978756a87539299d30.

Reply:
I further refined the language in 0582d45a56c3c8aca4ee670c4abd59a2fd215ec4.